docs/source/howto/_plot_x_y_databroker.ipynb

```diff
@@ -563,22 +590,123 @@
       "text/plain": [
-       "<Figure size 432x288 with 1 Axes>"
+       "<Figure size 640x480 with 1 Axes>"
       ]
      },
-     "metadata": {
-      "needs_background": "light"
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.plot(x.values, y.values)\n",
+    "plt.xlabel(x.name)\n",
+    "plt.ylabel(y.name)\n",
+    "plt.title(f\"scan_id={run.metadata['start']['scan_id']}\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d49133a8",
+   "metadata": {},
+   "source": [
+    "## 7. Plotting function\n",
+    "\n",
+    "Summarizing the code above, here is a handy function you can use to plot such $(x, y)$ data.  Or, you can use this code as a starting point to make your own plot function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "65470035",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "def xyplot(dataset, xname, yname, title=None):\n",
+    "    \"\"\"\n",
+    "    Plot the data from the primary stream (above).\n",
+    "\n",
+    "    Example::\n",
+    "\n",
+    "        xyplot(cat[-1].primary.read(), x.name, y.name)\n",
+    "    \"\"\"\n",
+    "    import matplotlib.pyplot as plt\n",
+    "\n",
+    "    x = dataset[xname]\n",
+    "    y = dataset[yname]\n",
+    "    title = title or f\"{yname} v. {xname}\"\n",
+    "\n",
+    "    plt.plot(x.values, y.values)\n",
+    "    plt.title(title)\n",
+    "    plt.xlabel(xname)\n",
+    "    plt.ylabel(yname)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "5f663bcf",
+   "metadata": {},
+   "source": [
+    "Here is an example using this function with the `dataset` from above.  The actual name for each data variable (must match the name provided by the `dataset`).  A plot title is provided."
```

Comment:
Why is "(must match the name provided by the `dataset`)" in parentheses? 
I usually try to avoid putting important information in parentheses. I would rather say:
"Here is an example using this function with the dataset from above. The actual name for each data variable must match the name provided by the dataset."

Reply:
agreed.